protocols/plaintext/src/lib.rs

```diff
@@ -18,15 +18,18 @@
 // FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
 // DEALINGS IN THE SOFTWARE.
 
-use futures::future::{self, Ready};
-use libp2p_core::{InboundUpgrade, OutboundUpgrade, UpgradeInfo, upgrade::Negotiated};
-use std::iter;
-use tokio_io::{AsyncRead, AsyncWrite};
-use tokio_io::codec::length_delimited::Framed;
 use crate::error::PlainTextError;
-use void::Void;
-use futures::future::FutureResult;
+use unsigned_varint::codec::UviBytes;
 use crate::handshake::Remote;
+use futures::future::{self, Ready};
+use futures::{Sink, Stream};
+use futures::prelude::*;
+use futures_codec::Framed;
+use libp2p_core::{identity, InboundUpgrade, OutboundUpgrade, UpgradeInfo, upgrade::Negotiated, PeerId, PublicKey};
```

Comment:
This line is much longer than 100 characters (cf. [.editorconfig](https://github.com/libp2p/rust-libp2p/blob/master/.editorconfig))